docs/prod-host-setup.md

```diff
@@ -64,7 +64,7 @@ Verification can be done by running:
 (grep -q "^forceoff$\|^notsupported$" /sys/devices/system/cpu/smt/control && echo "Hyperthreading: DISABLED (OK)") || echo "Hyperthreading: ENABLED (Recommendation: DISABLED)"
 ```
 
-#### Check Kernel Page-Table Isolation (KPTI) support
```

Comment:
Not sure if this is the best way to put it.
I would say [Intel only] and maybe add a phrase in the begining that platform specific info will be put in brackets...

Reply:
Done.